BuildAutoRest.ps1

```diff
@@ -229,6 +239,8 @@ ForEach ($SDK In $SDKName)
                     If (-not [System.String]::IsNullOrEmpty($UnusedTestFiles)) { $UnusedTestFiles | Remove-Item -Force -Recurse }
                     # Rebuild to distribute the update PSScriptInfo to other locations
                     $BuildModuleCommand = "$buildModulePath -Docs -Release"
+                    Write-Host ('[RUN COMMAND] Clean Script') -BackgroundColor:('Black') -ForegroundColor:('Magenta')
+                    bash $CleanScript $PSScriptRoot $OutputFullPath
```

Comment:
Before BuildModule is invoked we run the clean.sh script, it's the first step of buildModule but it'll fail if the permissions are not set correctly.